docs/user/performance.rst

```diff
@@ -13,80 +13,58 @@ Use magnitudes when possible
 
 It's significantly faster to perform mathematical operations on magnitudes (even though your'e still using pint to retrieve them from a quantity object).
 
-.. doctest::
+.. ipython::
+    :verbatim:
 
-   In [1]: from pint import UnitRegistry
+    In [1]: from pint import UnitRegistry
 
-   In [2]: ureg = UnitRegistry()
+    In [2]: ureg = UnitRegistry()
 
-   In [3]: q1 =ureg('1m')
+    In [3]: q1 = ureg('1m')
```

Comment:
there's [some](https://github.com/asottile/blacken-docs) [autoformatters](https://github.com/keewis/blackdoc) (the second one is mine) that apply `black` to code blocks in doctest lines or code blocks, which helps with keeping the code style consistent even in documentation.